oss-cloud-backend/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "oss-cloud-backend",
+  "version": "1.0.0",
+  "description": "Backend npm",
+  "main": "handler.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "serverless": "^1.50.1"
```

Comment:
Probably, you don't need serverless here at all.

Reply:
It shouldn't be part of 'dependencies' section, definitely. Let's move it to 'devDependencies'

Reply:
Serverless dependency is now in in devDependencies